ajax/libs/jquery-flexdatalist/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "jquery-flexdatalist",
+  "version": "1.9.4",
+  "description": "jQuery Datalist Autocomplete plugin",
+  "keywords": [
+    "jQuery",
+    "autocomplete",
+    "plugin",
+    "datalist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sergiodlopes/jquery-flexdatalist.git"
+  },
+  "author": "Sérgio Dinis Lopes",
+  "license": "MIT",
+  "homepage": "http://projects.sergiodinislopes.pt/flexdatalist/",
+  "npmName": "jquery-flexdatalist",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "**/*"
```

Comment:
It's a little bit not safe or no sense to use **/* under a non-dist folder. Let's use `jquery.flexdatalist.js*` instead.